modules/messages/server/controllers/messages.server.controller.js

```diff
@@ -303,8 +304,23 @@ exports.send = function(req, res) {
             return done(err);
           }
 
-          // Finally return saved message
-          return res.json(message);
+          // Finally return saved message...
+          res.json(message);
+
+          // ...and run agenda job to update sender and receiver reply rates
```

Comment:
When the sender's message is the _first reply_, her statistics need to change. (Her reply rate should go up.) Receiver's statistics don't need to change in this case.